packages/tsconfig/nextjs.json

```diff
@@ -13,6 +13,8 @@
     "noEmit": true,
     "incremental": true,
     "module": "esnext",
+    "moduleResolution": "Bundler",
```

Comment:
Very minor, for consistency with other settings (e.g. `esnext`): use `bundler` (lowercase)?